src/main/webapp/displaycharities.html

```diff
@@ -5,17 +5,31 @@
     <meta charset="UTF-8">
     <link rel="stylesheet" href="/css/main.css">
     <link rel="stylesheet" href="/css/user-page.css">
-    <script src="/js/display-charities.js"></script>
+    <script src="/js/events-suammry-loader.js"></script>
 </head>
-<body >
-<nav>
-    <ul id="navigation">
-        <li><a href="/">Home</a></li>
-        <li><a href="/aboutus.html">About Our Team</a></li>
-    </ul>
-</nav>
-<h1 id="page-title">Charities Page</h1>
```

Comment:
Can you please tell me more about why you changed this page? Isn't this the page Priyal created to display charities map/search? If that is the same page that Priyal created then these changes will completely make that page non-functional. Please let me know if this is not the case.

Reply:
This page displays all the events (user email, timestamp, UUID, title, description) and there is a form at the bottom of the page. Actually, I misnamed it. It is supposed to be display events instead. Sorry for the confusion.

Reply:
No problem, so as long as the name is  not the same as Priyal's file I will approve it. Just let me know :)